scripts/hostapd/certs/ca.cnf

```diff
@@ -0,0 +1,59 @@
+[ ca ]
+default_ca		= CA_default
+
+[ CA_default ]
+dir			= ./
+certs			= $dir
+crl_dir			= $dir/crl
+database		= $dir/index.txt
+new_certs_dir		= $dir
+certificate		= $dir/ca.pem
+serial			= $dir/serial
+crl			= $dir/crl.pem
+private_key		= $dir/ca.key
+RANDFILE		= $dir/.rand
+name_opt		= ca_default
+cert_opt		= ca_default
+default_days		= 365
+default_crl_days	= 30
+default_md		= md5
```

Comment:
I would rather use sha256 (given this is a prototype repo) and migrate down to sha1 in case we find any incompatibilities. Let's make strong security the default, not an afterthought.